IPython/terminal/interactiveshell.py

```diff
@@ -1,21 +1,569 @@
-# -*- coding: utf-8 -*-
-"""DEPRECATED: old import location of TerminalInteractiveShell"""
-
-# Copyright (c) IPython Development Team.
-# Distributed under the terms of the Modified BSD License.
+"""IPython terminal interface using prompt_toolkit in place of readline"""
```

Comment:
'in place of readline' is redundant now that we no longer have an equivalent class using readline ;-)